content/applications/finance/accounting/fiscal_localizations/localizations/chile.rst

```diff
@@ -674,13 +676,12 @@ Acceptation
 ~~~~~~~~~~~
 
 If all the commercial information is correct on your vendor bill, you can accept the document using
-the button: Aceptar Documento, once this is done the DTE Acceptation Status  changes to: Accepted and
-an email of acceptance is sent to the vendor. 
+the button: Aceptar Documento, once this is done the DTE Acceptation Status  changes to: Accepted
+and an email of acceptance is sent to the vendor. 
 
 .. image:: media/Chile35.png
```

Comment:
Hi @chiaraprattico 

Thanks for your review and comments. Regarding the images, actually, I compressed them all, but maybe I skipped one by mistake, could you tell me which pics are you specifically talking about to give them to @StraubCreative?

Thanks!

F.

Reply:
@fvz-odoo this was my mistake-- I needed to resize all of the images you gave me to HD resolution instead of where they were originally. 

We had some files that were approaching 4K at ~1920pixels when standard HD ~1280 is ideal. 

Will address on this next push.